src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SentenceCalculationService.kt

```diff
@@ -172,9 +175,14 @@ class SentenceCalculationService(
   }
 
   fun calculateFixedTermRecall(booking: Booking, days: Int): LocalDate {
-    return booking.returnToCustodyDate!!
-      .plusDays(days.toLong())
-      .minusDays(1)
+    try {
+      return booking.returnToCustodyDate!!
+        .plusDays(days.toLong())
+        .minusDays(1)
+    } catch (exception: Exception) {
```

Comment:
Not sure catching a NPE is the best. Also how would we ever get here now its validated up front?